src/main/kotlin/com/hibob/academy/employeeFeedback/resource/FeedbackResource.kt

```diff
@@ -34,15 +36,17 @@ class FeedbackResource(private val feedbackService: FeedbackService, private val
 
     @GET
     @Path("/allFeedback")
-    fun viewFeedback(@Context request: ContainerRequestContext): Response {
+    fun viewFeedback(@RequestBody filterBy: FeedbackFilterRequest, @Context request: ContainerRequestContext): Response {
         val employeeId = authenticatedUsersService.getLoggedInEmployeeId(request)
         val companyId = authenticatedUsersService.getLoggedInCompanyId(request)
         val role = authenticatedUsersService.getLoggedInRole(request)
         val validRoles = listOf(RoleType.ADMIN, RoleType.HR)
+        val anonymity = authenticatedUsersService.convertToAnonymityType(filterBy.anonymity)
```

Comment:
if you are using this in the view feedbacks as well, maybe the convertToAnonymityType shouldn't be inside the authenticatedUsersService.

The more right place for it is to be part of the enum